org.pwall.Pwall.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id": "org.pwall.Pwall",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "command": "pwall",
+    "modules": [
+      {
+          "name": "pwall",
+          "buildsystem": "simple",
+          "build-commands": [
+			"install -D flatpak/pwall-flatpak.desktop /app/share/applications/org.pwall.Pwall.desktop",
+      "install -D flatpak/org.pwall.Pwall.appdata.xml /app/share/metainfo/org.pwall.Pwall.appdata.xml",
+			"install -D res/pwall/pwall.png /app/share/icons/hicolor/128x128/apps/org.pwall.Pwall.png",
+		    "install -D res/pwall/help.html /app/share/help.html",
+		    "install -D res/pwall/noimage.png /app/share/noimage.png",
+			"install -D res/pwall/novideo.png /app/share/novideo.png",
+		    "install -D res/pwall/videoIcon.png /app/share/videoIcon.png",
+			"install -D res/pwall/clearbackup.sh /app/share/clearbackup.sh",
+		    "make -f makefile-flatpak",
+			"install -D pwall /app/bin/pwall",
+			"install -D flatpak/jhead /app/bin/jhead"
+            ],
+          "sources": [
+              {
+              "type": "git",
+              "url": "https://github.com/ltiber/pwall.git"
+             }    
+          ]
+      }
+  ],
+
+    "finish-args": [
+     "--socket=x11",
+     "--share=network",
+     "--filesystem=host",
```

Comment:
this is a very broad filesystem permission

Reply:
the main function of pwall is to be a photos and videos library. I need to have access to this file wherever they are.